src/style.css

```diff
@@ -1,3 +1,31 @@
 body {
-  background-color: bisque;
+  background-color: #f3e5e5;
```

Comment:
- [ ] I think it would be better if you use variables for colors because when you want to change a color it would be easier because you just change the variable and the change will appear in all instances of that variable.

So you can add this at the top of the css file
`
:root {
  --primary: #f3e5e5;
}`

and change line two of the style.css file like these
`background-color: var(--primary);`